stream-chat-android-ui-common/src/main/kotlin/io/getstream/chat/android/common/state/MessageAction.kt

```diff
@@ -81,6 +81,11 @@ public class Flag(message: Message) : MessageAction(message)
  */
 public class MuteUser(message: Message) : MessageAction(message)
 
+/**
+ * Block the user who sent the message.
+ */
+public class BlockUser(message: Message) : MessageAction(message)
```

Comment:
As mentioned, I think we can remove this from the default actions as it's behavior that's sample app specific. UI/UX wise, it doesn't make much sense to have user-based options when selecting messages.

Plus this behavior can be very tricky, as banning/blocking is highly dependant on app use cases